Bottom: 0be097b02838b7bfd6a8a14f960a2d8fcaded27a
Top:    098543f8afd604f23e885080b5eed39f1fcef222
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:37:01 +0000

Refresh of make-enable-dbus-default

---

diff --git a/configure.ac b/configure.ac
index c9b0dfe..7a4cc76 100644
--- a/configure.ac
+++ b/configure.ac
@@ -329,9 +329,9 @@ AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
 [],[
 	case " $with_gui " in
-		*\ gtk\ *)
+		*\ gtk\ *)     enable_dbus=yes ;;
 		*\ lesstif\ *) enable_dbus=yes ;;
-		* ) enable_dbus=no ;;
+		* )            enable_dbus=no ;;
 	esac
 ])
